src/components/MovieInfo/MovieInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Rating from '../Rating';
+import styles from './MovieInfo.scss';
+
+interface MovieInfoProps {
+  title: string;
+  genres: Array<{ id: number; name: string }>;
```

Comment:
please move it to the type or interface 
like a `export type GenresType = {
id: number; name: string
}`

then we can use it 
`genres: GenresType[]`